### DISCUSSION TITLE
[Deleted]

### DISCUSSION THREAD

**Discussion by nexensys** [ScratchAddons/ScratchAddons#2517] [Show and tell]
...

**Comment by TheColaber** [comment]
This is totally unrelated to scratch add-ons... Show and tell is for showing addon creations and cool things you've made for scratch add-ons.

**Reply by nexensys** [reply]
Okay I wil erase it then.

**Comment by cobaltt7** [comment]
This is better suited for the discord: https://discord.gg/Ak8sCDQ

**Reply by nexensys** [reply]
I can't use discord, sorry.

**Reply by retronbv** [reply]
> I can't use discord, sorry.

Why exactly can’t you?

**Reply by nexensys** [reply]
parents
